pkg/config/config.go

```diff
@@ -41,6 +42,37 @@ type Config struct {
 	Storage          *StorageConfig
 }
 
+// Load loads the config from a file.
+// If file is not present returns default config
+func Load(configFile string) (*Config, error) {
+	_, err := os.Stat(configFile)
+	if err == nil {
+		conf, err := ParseConfigFile(configFile)
+		return conf, err
+	} else if os.IsNotExist(err) {
+		log.Printf("config file not found under %s - using default settings", configFile)
+		return createDefault(), nil
+	}
+	return nil, err
+}
+
+func createDefault() *Config {
+	return &Config{
+		GoBinary:         "go",
+		GoEnv:            "development",
+		GoGetWorkers:     30,
+		ProtocolWorkers:  30,
+		LogLevel:         "debug",
+		CloudRuntime:     "none",
+		TimeoutConf:      TimeoutConf{Timeout: 300},
+		StorageType:      "memory",
+		Port:             ":3000",
+		GlobalEndpoint:   "http://localhost:3001",
+		TraceExporterURL: "http://localhost:14268",
```

Comment:
Would you not want to specify type as jaeger if you have an export url?

Reply:
Yep! Actually, I am thinking that we should remove all parameters related to observability now from dev.